tests/org.jboss.tools.openshift.test/src/org/jboss/tools/openshift/test/core/server/OpenShiftServerUtilsTest.java

```diff
@@ -246,4 +249,74 @@ public void should_not_show_jmx_on_non_java_project() throws CoreException {
 	    doReturn(eclipseProject.getName()).when(server).getAttribute(eq(OpenShiftServerUtils.ATTR_DEPLOYPROJECT), anyString());
 	    assertThat(OpenShiftServerUtils.isJavaProject(server)).isFalse();
 	}
+
+	@Test
+	public void shouldFindServerGivenListOfServersContainsServerWithNullType() throws CoreException {
+		// given
+		IServer os3Server = mockOS3Server("os3", "someService");
```

Comment:
this is a test, there's no added value in defining a constant to make sure we're always using the very same string.